Azure/azure-cli
# [WebPubSub] Initial PR for Azure WebPubSub Service

**Description**

Initial PR for onboard azcli for Azure WebPubSub Service

**Testing Guide**

General commands:
Create: `az webpubsub create`
Update: `az webpubsub update`
Show: `az webpubsub show`
List: `az webpubsub list`
Restart: `az webpubsub restart`
Delete : `az webpubsub delete`

Key management:
List keys: `az webpubsub key list`
Regenerate keys: `az webpubsub key regenerate`

Event handler management:
Clear all event handler: `az webpubsub event-handler clear`
General update event handler: `az webpubsub event-handler update`
List event handler: `az webpubsub event-handler list`
Update with subgroup command: `az webpubsub event-handler hub update` `az webpubsub event-handler hub remove`

Network rules management:
List rules: `az webpubsub network-rule list`
Update rules: `az webpubsub network-rule update`
---

This checklist is used to make sure that common guidelines for a pull request are followed.
